server.js

```diff
@@ -0,0 +1,8 @@
+const app = require('./lib/app');
+const http = require('http');
+
+
+http
```

Comment:
```suggestion
const PORT = 5000;

http
    .createServer(app)
    .listen(PORT, () => console.log('sever Listening on port', PORT));
```
Another way to craft your server.js file in preparation for a .env file :)